content/en/opentelemetry/correlate/dbm_and_traces.md

```diff
@@ -0,0 +1,111 @@
+| `db.name`      | The logical database or schema name being queried.                                                  | `user_accounts`                    |
+| `span.type`    | **Required (Datadog-specific).** The type of span such as `sql`,`postgres`, `mysql`, or `sql.query` | `sql`                              |
+
+#### Example
+
+The method for adding these attributes depends on your setup. If you are using an OpenTelemetry auto-instrumentation library for your database client, see its documentation for configuration options. If you are manually creating spans with the OpenTelemetry SDK, you can set the attributes directly in your code. For more information, see the [OpenTelemetry documentation][4].
+
+The following is a conceptual example of manual instrumentation using Python's OpenTelemetry SDK:
+
+```python
+from opentelemetry import trace
+
+tracer = trace.get_tracer("my-app.instrumentation")
+
+# When making a database call, create a span and set attributes
+with tracer.start_as_current_span("postgres.query") as span:
+    # Set attributes required for DBM correlation
+    span.set_attribute("span.type", "sql")
+    span.set_attribute("db.system", "postgres")
+    span.set_attribute("db.statement", "SELECT * FROM users WHERE id = ?")
+    span.set_attribute("db.name", "user_accounts")
+
+    # Your actual database call would go here
+    # db_cursor.execute("SELECT * FROM users WHERE id = %s", (user_id,))
+```
+
+### Step 2: Configure your ingest path
+
+Depending on how you send traces to Datadog, you may need to enable specific feature gates to ensure database spans are processed correctly.
+
+{{< tabs >}}
+{{% tab "Datadog Agent (DDOT Collector)" %}}
+
+
+If you are using the Datadog Helm chart (v3.107.0 or later), set the feature gate in your `values.yaml`:
+
+```yaml
+datadog:
+  otelCollector:
+    featureGates: datadog.EnableOperationAndResourceNameV2
+```
+
+{{% /tab %}}
+{{% tab "OTel Collector" %}}
+
+When starting the Collector, enable the `datadog.EnableOperationAndResourceNameV2` feature gate. This is available in Collector v0.118.0 and later.
+
+```sh
+otelcontribcol --config=config.yaml \
+--feature-gates=datadog.EnableOperationAndResourceNameV2
+```
+
+{{% /tab %}}
+
+{{% tab "Datadog Agent (OTLP Ingest)" %}}
+
+In your Datadog Agent configuration, ensure the `DD_APM_FEATURES` environment variable includes `enable_operation_and_resource_name_logic_v2`.
+
+{{% /tab %}}
+
+{{< /tabs >}}
+
+### View correlated data in Datadog
+
+After your application is sending traces, you can see the correlation in the APM Trace View:
+
+1. Navigate to [**APM** > **Traces**][3].
+2. Find and click on a trace from your instrumented service.
+3. In the trace's flame graph, select a database span (for example, a span with `span.type: sql`)
+4. In the details panel, click the **SQL Queries** tab. You should see the host metrics, like CPU and memory utilization, from the host that executed that part of the request.
+
+## Further reading
```

Comment:
Do we want to have a list of some known gotchas at this point? The correlation here is best-effort and not guaranteed, so I want to make sure people are aware of that.